Routers/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿using NUnit.Framework;
+using Routers;
+
+ReadFile file = new ReadFile("text.txt");
```

Comment:
```suggestion
ReadFile file = new("text.txt");
```
Не пишите один тип дважды